brage-import/src/test/java/no/sikt/nva/brage/migration/merger/PublicationInstanceMergerTest.java

```diff
@@ -68,7 +85,7 @@ void shouldUseExistingPublicationInstanceWhenNewPublicationInstanceIsEmpty(Publi
         var updatedPublication = mergePublications(existingPublication, bragePublication);
 
         assertThat(updatedPublication.getEntityDescription().getReference().getPublicationInstance(),
-                   doesNotHaveEmptyValues());
+                   doesNotHaveEmptyValuesIgnoringFields(Set.of("duration")));
```

Comment:
Add the field to ignoredFields.txt instead, that way we keep all our fields that are exempt for doesNotHaveEmptyValues in one place.

Reply:
Have added, but it is a test, this test will still fail if I just do doesNotHaveEmptyValues(), cause random publication instance containing "duration" is always null.